app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -56,6 +56,7 @@ def perform(_now)
   attr_accessor :enrollment_outcomes
 
   DEFAULT_EMAIL_DELAY_IN_HOURS = 1
+  DEFAULT_SMS_DELAY_IN_HOURS = 1
```

Comment:
is there ever a time when we'd want the delay to differ from the delay for emails? if not could we use a singular default value named something like `DEFAULT_MESSAGE_DELAY_IN_HOURS`?